src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+body {
```

Comment:
shouldn't be here

Reply:
It's the general styles, that should be in the index.scss
<img width="634" alt="image" src="https://github.com/user-attachments/assets/0a9d7862-45a6-4591-b124-9b1610ac2a9e">